src/System.ComponentModel.Composition.Registration/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
-﻿<Project>
+<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
Shouldn't this be bumped and then the ref assembly leave it as 4.0.0.0 in order to match full framework?

Reply:
Yes, in cases where we still unify with full framework and ship a package we *freeze* the ref and then keep incrementing the implementation.